templates/configuration/graphdb-setting-configmap.yaml

```diff
@@ -0,0 +1,15 @@
+# Default configuration map for provisioning GraphDB repository.
+# To change it, prepare another configuration map and update "graphdb.repositoryConfigmap"
+{{- $configs := (.Values.graphdb.configs | default dict) }}
+{{- $settingsConfigMap := $configs.settingsConfigMap | default "null"}}
+{{- if or (eq $settingsConfigMap "graphdb-settings-configmap") (and (eq $settingsConfigMap "null") (.Values.graphdb.security.enabled)) }}
```

Comment:
Is there a better way to check for the nullability of Values.graphdb.configs.settingsConfigMap ?

Reply:
I'm still reading into it but it's tricky with every statement in the if being evaluated and prone to giving out errors

Reply:
changed it to be slightly cleaner, can't think of a better way that doesn't include multiple ifs that lead to the same result